python/uagents-core/uagents_core/storage.py

```diff
@@ -32,23 +32,37 @@ def compute_attestation(
 
 
 class ExternalStorage:
-    def __init__(self, identity: Identity, storage_url: Optional[str] = None):
+    def __init__(
+        self,
+        identity: Optional[Identity] = None,
+        storage_url: Optional[str] = None,
+        api_token: Optional[str] = None,
+    ):
         self.identity = identity
+        self.api_token = api_token
         self.storage_url = storage_url or AgentverseConfig().storage_endpoint
 
     def _make_attestation(self) -> str:
         nonce = token_bytes(32)
-        now = datetime.utcnow()
+        now = datetime.now()
```

Comment:
Is it okay that this datetime does not include timezone? I guess that agentverse would verify it using UTC?